src/Umbraco.Cms.Integrations.Automation.Zapier/Migrations/ZapContentConfigTable.cs

```diff
@@ -1,30 +1,62 @@
 ﻿using NPoco;
 
+#if NETCOREAPP
+using Microsoft.Extensions.Logging;
+
+using Umbraco.Cms.Infrastructure.Migrations;
+using Umbraco.Cms.Infrastructure.Persistence.DatabaseAnnotations;
+#else
 using Umbraco.Core.Migrations;
 using Umbraco.Core.Persistence.DatabaseAnnotations;
 using Umbraco.Core.Logging;
+#endif
 
 namespace Umbraco.Cms.Integrations.Automation.Zapier.Migrations
 {
     public class ZapContentConfigTable : MigrationBase
     {
+        public string MigrationLoggingMessage = $"Running migration {Constants.MigrationPlanName}";
+
+        public string DbTableExistsMessage =
+            $"The database table {Constants.ZapContentConfigTable} already exists, skipping";
+
         public ZapContentConfigTable(IMigrationContext context) : base(context)
         {
         }
 
+#if NETCOREAPP
```

Comment:
You perhaps only need the compiler directive around the method signature and the logger, which means you don't need to repeat the actual migration logic itself.